vmdb/app/controllers/application_controller/ci_processing.rb

```diff
@@ -1902,6 +1902,7 @@ def show_association(action, display_name, listicon, method, klass, association
       @x_show = params[:x_show]
       @sb[:action] = @lastaction = action
     end
+    set_config(identify_record(params[:id]))
```

Comment:
@bmclaughlin since "show_association" is a common method, and this call is specific to host record, it should be done in host controller.